microsoft-edge/extensions-chromium/publish/aad-account.md

```diff
@@ -0,0 +1,67 @@
+---
+description: Learn how to add and manage users from your organization on to the Microsoft Edge program
+title: Register your organization for Microsoft Edge program on Partner Center
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 07/27/2021
+ms.topic: conceptual
+ms.prod: microsoft-edge
+keywords: edge-chromium, extensions development, browser extensions, add-ons, partner center, developer
+---
+
+# Add and manage users from your organization on to the Microsoft Edge program
+
+To add and manage users in the Microsoft Edge program to manage extensions, you can now associate your Partner Center account with your organization's Azure Active Directory.
+
+## Step 1: Enroll in the Microsoft Edge program on Partner Center 
+
+1. If you are new to Partner Center and don't have an account or if you have an existing commercial account with Partner Center (see step **2.c** below), create a new [Microsoft account (MSA)][WindowsCommunityEverythingAboutMicrosoftAccounts] (or use an existing one) to register with the Microsoft Edge program. Follow the steps in the [Enroll in the Microsoft Edge program for Partner Center][DeveloperRegistration]. 
```

Comment:
Can we change the "Enroll in the Microsoft Edge..." link text to the H1 of the linked page?

Reply:
"Create a new Microsoft account (MSA)" > can we change to "For more information on ..."